src/api_calls/orders_calls.js

```diff
@@ -50,7 +50,7 @@ export async function getAllOrders()
         });
 }
 
-export function deleteOrder(id){
+export async function deleteOrder(id){
```

Comment:
It is not necessary to declare these function `async` if you do not use `await` in function block.